example/pubspec.lock

```diff
@@ -86,57 +97,66 @@ packages:
   flutter_inappwebview:
     dependency: transitive
     description:
-      name: flutter_inappwebview
-      url: "https://pub.dartlang.org"
-    source: hosted
+      path: "."
+      ref: "heads/v5.7.2+3_fix_xcode_17"
+      resolved-ref: "2d6c123b8065cdd98593b783a811e79d433e7244"
+      url: "https://github.com/nesquikm/flutter_inappwebview"
+    source: git
```

Comment:
https://pub.dev/packages/flutter_inappwebview

Seems like you can use the package again, since they updated it to xcode 17